test/performance/observed_concurency_test.go

```diff
@@ -198,8 +199,11 @@ func TestObservedConcurrency(t *testing.T) {
 			tc = append(tc, CreatePerfTestCase(float32(toConcurrency/time.Millisecond), fmt.Sprintf("to%d", i), tName))
 		}
 	}
+	// TODO: For future, recreate the CreateTestgridXML function so we don't want to repeat the code shown in next 2 lines
+	ts := junit.TestSuites{}
+	ts.AddTestSuite( &junit.TestSuite{Name:"TestPerformanceLatency", TestCases:tc} )
 
-	if err = testgrid.CreateTestgridXML(tc, tName); err != nil {
+	if err = testgrid.CreateXMLOutput(&ts, "TestPerformanceLatency"); err != nil {
```

Comment:
```suggestion
	if err = testgrid.CreateXMLOutput(&ts, tName); err != nil {
```

Reply:
@adrcunha , @srinivashegde86 , @nbarthwal , I think the XML error is caused by this line

Reply:
yeah. it seems that latency and observed concurrency tests are both writing into the same file. We should just be using tName here instead that points to the test name